internal/rbd/rbd_util.go

```diff
@@ -552,6 +554,16 @@ func genVolFromVolID(ctx context.Context, volumeID string, cr *util.Credentials,
 		}
 	}
 
+	rbdVol.ImageID, err = j.GetReservedImageID(ctx, rbdVol.JournalPool, rbdVol.ReservedID, cr)
+	if err != nil {
+		// skip Not found error for older RBD images, as we have not saved
```

Comment:
should we insert the image id here? might make it easier for upgrades? file an issue and mark as TODO?

Reply:
it's already handled in deleteVolume, if required later we can move it here also

Reply:
no idea if it is needed, maybe others can recommend (against) it?